packages/instant/src/components/instant_heading.tsx

```diff
@@ -9,17 +9,20 @@ import { format } from '../util/format';
 
 import { AmountPlaceholder } from './amount_placeholder';
 import { Container, Flex, Text } from './ui';
+import { Icon } from './ui/icon';
 
 export interface InstantHeadingProps {
     selectedAssetAmount?: BigNumber;
     totalEthBaseAmount?: BigNumber;
     ethUsdPrice?: BigNumber;
     quoteRequestState: AsyncProcessState;
+    buyOrderState: AsyncProcessState;
 }
 
 const placeholderColor = ColorOption.white;
 export class InstantHeading extends React.Component<InstantHeadingProps, {}> {
     public render(): React.ReactNode {
+        const iconOrAmounts = this._renderIcon() || this._renderAmountsSection();
```

Comment:
I believe we need to bind `this` here OR make `_renderIcon` and `_renderAmountsSection` arrow functions, e.g.

```
private readonly _renderAmountsSection = () => {
        return (
            <Container>
                <Container marginBottom="5px">{this._placeholderOrAmount(this._ethAmount)}</Container>
                <Container opacity={0.7}>{this._placeholderOrAmount(this._dollarAmount)}</Container>
            </Container>
        );
    }
```
See `buy_button.tsx`

I think we prefer the arrow function approach when in a render method for performance reasons, a new function does not need to be created for each render cycle

Reply:
Per convo, since this doesn't need to be bound, current implementation is fone

Reply:
private instance methods like this feel more natural to me, would prefer to leave

Reply:
although I could see the appeal of not having to think about this

Reply:
I personally always do arrow functions just so I don't have to think about it but don't feel strongly about it.